### PR TITLE
new player cards

### PR DESCRIPTION
**Pull request by hlyford11**
- and stadium images

### CODE REVIEW

File: views/partials/nhl_player_card [hunk 1]
```diff
@@ -0,0 +1,41 @@
+
+<script id="card" type="text/x-handlebars-template">
```
Comment (blakeparkinson) [inline views/partials/nhl_player_card]: what's the point of making this card if it's just a copy/paste of the nba card? We should just add if clauses in the player_card when needed or do another subtemplate where things really start diverging

Comment (HenryEdmodo) [inline views/partials/nhl_player_card]: it's not done yet. some items wil likely change. I thoguth it was worth it for easier organizing even if not quite as modular.

Comment (blakeparkinson) [inline views/partials/nhl_player_card]: the only thing that will change are whatever the stats are called right? We should just do a a sub sub template in this sub template

Comment (hlyford11) [inline views/partials/nhl_player_card]: that is WAY too meta for me.

On Wed, Apr 8, 2015 at 5:09 PM, Blake Parkinson <notifications@github.com>
wrote:

> In views/partials/nhl_player_card
> <https://github.com/blakeparkinson/sports2/pull/243#discussion_r28026300>:
>
> > @@ -0,0 +1,41 @@
> > +
> > +<script id="card" type="text/x-handlebars-template">
>
> the only thing that will change are whatever the stats are called right?
> We should just do a a sub sub template in this sub template
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blakeparkinson/sports2/pull/243/files#r28026300>.
>



-- 
*Henry Lyford*
Manager, Partnerships & Business Development
Edmodo.com/henry

**Pull request merged**